hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/fs/AbstractSpaceUsageSource.java

```diff
@@ -86,6 +86,7 @@ public long getAvailable() {
 
   @Override
   public long getCapacity() {
-    return file.getTotalSpace();
+    // system reserve: fsFree - fsUsable
+    return file.getTotalSpace() - (file.getFreeSpace() - file.getUsableSpace());
```

Comment:
Through I understand that there is different between getFreeSpace and getUsableSpace, I would suggest not change this, for we would like to know the full capacity of the volume.

Reply:
Updated, PTAL.